cluster/engine.go

```diff
@@ -116,6 +116,7 @@ type Engine struct {
 func NewEngine(addr string, overcommitRatio float64, opts *EngineOpts) *Engine {
 	e := &Engine{
 		Addr:            addr,
+		Name:            "(unknown)",
```

Comment:
It's not clear to me how there can be side effects, but perhaps it makes sense to do this in Info anyway, since other display changes for pending engines (if any) will be in info.